streams/shared/src/main/scala/zio/stream/ZTransducer.scala

```diff
@@ -540,6 +540,18 @@ object ZTransducer {
         }
     }
 
+  /**
+   * Decodes chunks of ISO/IEC 8859-1 bytes into strings.
+   *
+   * This transducer uses the String constructor's behavior when handling malformed byte
+   * sequences.
+   */
+  val iso8859_1: ZTransducer[Any, Nothing, Byte, String] =
```

Comment:
shall we call it iso8859_1Decode to be aligned with utf8Decode? And allow for a potential encoding transducer in the future.